docs/flows.md

```diff
@@ -23,16 +28,16 @@ Input and output schemas for flows can be defined using `zod`.
 import { defineFlow } from '@genkit-ai/flow';
 import * as z from 'zod';
 
-export const myFlow = defineFlow(
+export const menuSuggestionFlow = defineFlow(
   {
-    name: 'myFlow',
-    inputSchema: z.object({ subject: z.string() }),
+    name: 'menuSuggestionFlow',
+    inputSchema: z.object({ cuisine: z.string() }),
     outputSchema: z.string(),
   },
   async (input) => {
-    const output = doSomethingCool(input.subject);
+    const suggestion = makeMenuSuggestion(input.cuisine);
```

Comment:
Just the input get wrapped in an `input` object? Just trying to reconcile some examples here have it and others do not.

Reply:
Yeah, but on second thought this seems like overkill. Just gonna do `z.string()` for simplicity.